Update Baseline glossary entry (#26779)
mdn/content: files/en-us/glossary/baseline/compatibility/index.md

**Baseline** identifies widely-supported web platform features.
Web developers can expect Baseline features to work in contemporary browsers.

> A Baseline feature - such as an API, a set of CSS properties, or a JavaScript syntax - is known to work consistently across two or more major versions of all of several well-known browsers (Chrome, Edge, Firefox, and Safari).

Reviewer: Thanks for the clarification, Arka! I'd say "is known to work consistently in at lest the current and previous major versions ..."

Reviewer 2: Exactly this wording is why I opened an issue :)

`current and previous major versions` doesn't state the number of versions.
That can be misunderstood/misread as `5 versions`, `10 versions`, ....

Reviewer 3: "At least one version before the current version"? TBH I don't think it has to be too rigorous...

Reviewer 2: There has been a lot of feedback on `last 2 majors` for Baseline, with most (all?) feedback in the direction of `this is not enough, needs to be more than last 2 majors`.

With that in mind I do think it is important that it is correctly and unambiguously documented that Baseline features are possibly only supported in 2 major versions at a given point in time.

I personally do not see the benefit of choosing a wording that is more open to interpretation here :)

Author: I believe we all have to come to one particular wording / textual expression which everyone agrees upon.

Author: Requesting every participant in this conversation to come to a conclusion, so that we can complete this issue.

Reviewer 3: I believe everyone here has settled on "At least one version before the current version".

Author: Updated the PR.

Author: Can this be merged ?

> A Baseline feature - such as an API, a set of CSS properties, or a JavaScript syntax - is known to work consistently in the current and the previous major version of several well-known browsers (Chrome, Edge, Firefox, and Safari).

Reviewer 4: I'm somewhat conflicted about this change. For one I'm not sure it's meaningfully different. Plus, I think it's nice to emphasize the plurality of releases involved here (i.e., it takes no less than 8 releases and very likely more to achieve Baseline). What do you think of something like this?

```suggestion
A Baseline feature - such as an API, a set of CSS properties, or a JavaScript syntax - is known to work consistently across two or more major versions of all of several well-known browsers (Chrome, Edge, Firefox, and Safari).
```

Or wordier, but arguably less complex to parse:

```suggestion
A Baseline feature - such as an API, a set of CSS properties, or a JavaScript syntax - is known to work consistently in all of several well-known browsers (Chrome, Edge, Firefox, and Safari) for two or more major releases of each browser.
```

That said, I'm reluctant to get _too_ legalistic about the definition here. There are going to be opportunities for editorial overrides—e.g., for deprecation or severe bugs—and the definition is likely to change, based on the feedback we've received so far.

Author: Hi @romainmenke , what would be your feedback on this ?

Reviewer 2: Either is fine with me.

They at least communicate that the minimum of Baseline really is just two majors and not an unknown number of versions.

Author: Then, I believe we have to wait till the code owner approves it. Thanks 😃

Reviewer 3: @postmeback You just have to commit the change above, because Daniel _is_ the person that contributed this definition in https://github.com/mdn/content/pull/26676.

Author: I updated the PR. Please re-review it. 😃

> A Baseline feature - such as an API, a set of CSS properties, or a JavaScript syntax - is known to work consistently at least one version before the current version of all of several well-known browsers (Chrome, Edge, Firefox, and Safari).

Reviewer 2: ```suggestion
A Baseline feature - such as an API, a set of CSS properties, or a JavaScript syntax - is known to work consistently in at least one version before the current version of all of several well-known browsers (Chrome, Edge, Firefox, and Safari).
```

typo :)
